geschaeftsordnung-vollversammlung.md

```diff
@@ -0,0 +1,82 @@
+
+# Geschäftsordnung der Vollversammlung der Fachschaft Digital Engineering
+
+Diese Geschäftsordnung wurde am xx.xx.xxxx von der Vollversammlung beschlossen.
+
+## § 1 Geltungsbereich
+
+Die Geschäftsordnung gilt für die Vollversammlung der Fachschaft Digital Engineering an der Universität Potsdam.
+
+## § 2 Tagesordnung
+
+(1) Die Vollversammlung folgt einer Tagesordnung.
+
+(2) Die Tagesordnung muss Teil der fristgerechten Einladung einer Vollversammlung sein.
+
+(3) Änderungsanträge zur Tagesordnung einer ordentlichen Vollversammlung können bis zu 14 Tage vor der Vollversammlung bei den Einladenden gestellt werden.
+
+(4) 7 Tage vor einer ordentlichen Vollversammlung muss die finale Tagesordnung auf dem gleichen Weg wie die Einladung zur Vollversammlung bekanntgegeben werden.
+
+## § 3 Versammlungsleitung
+
+(1) Die Vollversammlung wird von den Einladenden geleitet.
+
+(2) Die Versammlungsleitung kann für einzelne Tagesordnungspunkte an andere Mitglieder der Fachschaft übergeben werden.
+
+(3) Die Versammlungsleitung kann bei Bedarf eine Redeliste führen und / oder die Redezeit begrenzen.
+
+(4) Die Versammlungsleitung entscheidet über die explizite Zählung und Protokollierung von Stimmen.
+
+## § 4 Protokolle
+
+(1) Die Vollversammlung dokumentiert ihre Beschlüsse in einem Protokoll.
+
+(2) Die Versammlungsleitung beauftragt ein Mitglied der Fachschaft mit der Protokollführung.
+
+(3) Das Protokoll wird der Fachschaft spätestens 7 Tage nach der Vollversammlung zugänglich gemacht.
+
+## $ 5 Anträge
+
+Anträge können vor der Behandlung durch die antragstellende Person zurückgezogen werden.
+
+## § 6 Änderungsanträge
+
+(1) Änderungsanträge sind Anträge zur Änderung der Tagesordnung oder zur Änderung von Anträgen.
```

Comment:
mMn ist das eine verwirrende und unübliche Terminologie. Ich würde Änderungsanträge nur zu einem Antrag ermöglichen, die Tagesordnung kann ja entweder im Vorfeld durch unbürokratische Kommunikation mit den Antragstellenden oder per GO-Antrag geändert werden

Reply:
@jeriox Mit einem Änderungsantrag auf eine TO bezogen meine ich vermutlich das gleiche wie du mit einem gewöhnlichen Antrag